src/Workspaces/Core/Desktop/SymbolSearch/SymbolSearchUpdateEngine.Update.cs

```diff
@@ -263,12 +302,9 @@ private string ConvertToFileName(string source)
                 catch (Exception e) when (_service._reportAndSwallowException(e))
                 {
                     // We retrieved bytes from the server, but we couldn't make a DB
-                    // out of it.  That's very bad.  Just trying again one minute later
-                    // isn't going to help.  We need to wait until there is good data
-                    // on the server for us to download.
-                    var failureDelay = _service._delayService.CatastrophicFailureDelay;
-                    await _service.LogInfoAsync($"Unable to create database from full database element. Update again in {failureDelay}").ConfigureAwait(false);
-                    return failureDelay;
+                    // out of it.  That's very bad.  We'll have to try again later.
+                    await _service.LogInfoAsync($"Unable to create database from full database element.").ConfigureAwait(false);
+                    throw;
```

Comment:
No.  There is a try/catch one up in the caller that handles this and appropriate notifies the host (VS) that we werne't successful getting the full index.  This will manifest as an appropriate message in the new VS task-list.